apps/src/templates/manageStudents/ConfirmRemoveStudentDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes} from 'react';
+import i18n from '@cdo/locale';
+import {Header, ConfirmCancelFooter} from '../../lib/ui/SystemDialog/SystemDialog';
+import BaseDialog from '../BaseDialog';
+import Button from "../Button";
+import color from "../../util/color";
+
+export default class ConfirmRemoveStudentDialog extends React.Component {
+  static propTypes = {
+    isOpen: BaseDialog.propTypes.isOpen,
+    disabled: PropTypes.bool,
+    onConfirm: PropTypes.func.isRequired,
+    onCancel: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {isOpen, disabled, onConfirm, onCancel} = this.props;
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        isOpen={isOpen}
+        handleClose={onCancel}
+      >
+        <div style={styles.container}>
+          <Header text={i18n.removeStudentHeaderNew()}/>
+          <p>
+            <strong>{i18n.removeStudentBody1()}</strong>
+            {' '}
+            {i18n.removeStudentBody2()}
+          </p>
+          <p>
+            {i18n.removeStudentBody3()}
+          </p>
+          <Button
+            text={i18n.removeStudentSendHomeInstructions()}
+            target="_blank"
+            href="https://support.code.org/hc/en-us/articles/115001475131-Adding-a-personal-login-to-a-teacher-created-account"
```

Comment:
we're going to have this link in a few places with the new account deletion flow. is there a shared place we can put it?

Reply:
How about https://github.com/code-dot-org/code-dot-org/pull/23619/commits/677f0c7063b4ae33e06b540fa713f229be29cad1 ?

Reply:
perfect! 🎉